### PR TITLE
Fixes #771 Replace hardcoded strings with string resource in the project

### LINKED ISSUE #771: Replace hardcoded strings with string resource in the project
Hardcoded strings are used at some places in the project. It would be great if we replace the strings with string resource.

I would like to work on the issue.

### PR DESCRIPTION
**Pull request by AdityaGupta030697**
## Description
Replaced the strings with string resource in XML files and updated the strings.xml.

## Related issues and discussion
#771

### CODE REVIEW

**Comment by policorn** [comment]
The typo in the PR title made my day xD

**Comment by AdityaGupta030697** [comment]
sorry for that  :sweat_smile:

**Comment by AdityaGupta030697** [comment]
sure :)

File: app/src/main/res/layout/fragment_find_product.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
         android:hint="@string/hintBarcode"
         android:inputType="number"
         android:maxLines="1"
-        android:paddingLeft="10dip" />
+        android:paddingLeft="10dp" />
```
Comment (policorn) [inline app/src/main/res/layout/fragment_find_product.xml]: Still hardcoded I'm afraid

File: app/src/main/res/layout/nutriment_item_list.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
         android:layout_marginLeft="@dimen/activity_horizontal_margin"
         android:layout_marginRight="@dimen/activity_horizontal_margin"
         android:gravity="end"
-        tools:text="496 Kcal"
+        tools:text="@string/kcal_496"
```
Comment (policorn) [inline app/src/main/res/layout/nutriment_item_list.xml]: Probably is better to use a more generic name? I also think the Top/Bottom margins (for example) can be also referenced instead or being hardcoded

**Comment by policorn** [comment]
Not sure of the issue #771's scope. If it's intended to clean up the resources/layouts or the app and tests files as well.

File: app/src/main/res/layout/nutriment_item_list.xml (hunk 2 above)
Comment (AdityaGupta030697) [reply app/src/main/res/layout/nutriment_item_list.xml]: sure

**Comment by AdityaGupta030697** [comment]
@policorn  
Sir,
My aim is to replace hardcoded text with string resource.

**Pull request closed without merging**